runtime/autoload/provider/node.vim

```diff
@@ -5,35 +5,53 @@ let g:loaded_node_provider = 1
 
 let s:job_opts = {'rpc': v:true, 'on_stderr': function('provider#stderr_collector')}
 
+function! s:is_minimum_version(version, min_major, min_minor) abort
+  let nodejs_version = a:version
+  if !a:version
+    let nodejs_version = get(split(system(['node', '-v']), "\n"), 0, '')
+    if v:shell_error || nodejs_version[0] !=# 'v'
+      return 0
+    endif
+  endif
+  " [major, minor, patch]
+  let v_list = !!a:version ? a:version : split(nodejs_version[1:], '\.')
```

Comment:
What's the ternary for if `a:version` is string output of `node -v`?

Reply:
Yeah that was a goof.